server/controllers/article_controller.js

```diff
@@ -11,8 +11,8 @@ export const createArticle = (uri, groups, score) => {
   const article = new Article();
   article.uri = uri;
   article.groups = groups;
-  article.avgUserScore = (score || 1);
-  article.numShares = (score ? 1 : 0);
+  article.avgUserScore = (score ? score : 1);
```

Comment:
oh is that why that changed? I seemed to have some trouble where it was always evaluating as 1 and thought that was because score || 1 always evaluates to true so it takes 1? idk